### ISSUE TITLE
Problem with transformer after Jest @28.x.x release

### ISSUE THREAD

**Issue by PrzemekZygmanowski** [ambar/rollup-jest#22]
Hello,
After release  of newest Jest version 28 process() and processAsync() methods of a custom [transformer module] cannot return a string anymore. They must always return an  object or a Promise resolving to an object. The object must have `code` property with a string of processed code.

here is a source:
https://jestjs.io/docs/upgrading-to-jest28#transformer

**Comment by ambar** [comment]
I updated jest to v28 in the main branch, and it's compatible for now.

**Comment by PrzemekZygmanowski** [comment]
Hello, Your changes didn't change anything:
   ● Invalid return value:
      `process()` or/and `processAsync()` method of code transformer found at 
      "C:/xxx/node_modules/rollup-jest/transform.js" 
      should return an object or a Promise resolving to an object. The object 
      must have `code` property with a string of processed code.
      
    maybe You should add `code` object after return and update the version of the package?

**Comment by ambar** [comment]
I mean is the current code (man branch) looks compatible with Jest 28 (macOS/Linux), do you mean it is not compatible with your env?

**Comment by PrzemekZygmanowski** [comment]
no it's not compatible. 
I have installed jest 28and after launch and after launching the tests, I get an error 
```
● Invalid return value:
process() or/and processAsync() method of code transformer found at
"C:/xxx/node_modules/rollup-jest/transform.js"
should return an object or a Promise resolving to an object. The object
must have code property with a string of processed code.
```

**Comment by ambar** [comment]
@PrzemekZygmanowski Released v2.0, should be fixed.
